app/components/styles-docs/StyleUtilitiesSamples.js

```diff
@@ -1039,6 +1061,166 @@ export const CLASS_SPECIFIC_SAMPLE_COMPONENT = {
       );
     },
   },
+  MARGIN__ENDS_WITH___N1: {

```

Comment:
unnecessary margin related code added here

Reply:
the unrelated code of margin was removed as it is not related to this particular issue.